src/raiden_libs/blockchain.py

```diff
@@ -43,25 +43,11 @@ def create_registry_event_topics(contract_manager: ContractManager) -> List:
     return [encode_hex(event_abi_to_log_topic(new_network_abi))]
 
 
-def decode_event(abi: ABI, log_: Dict) -> Dict:
-    """ Helper function to unpack event data using a provided ABI
-
-    Args:
-        abi: The ABI of the contract, not the ABI of the event
-        log_: The raw event data
-
-    Returns:
-        The decoded event
-    """
-    if isinstance(log_["topics"][0], str):
-        log_["topics"][0] = decode_hex(log_["topics"][0])
-    elif isinstance(log_["topics"][0], int):
-        log_["topics"][0] = decode_hex(hex(log_["topics"][0]))
-    event_id = log_["topics"][0]
-    events = filter_by_type("event", abi)
-    topic_to_event_abi = {event_abi_to_log_topic(event_abi): event_abi for event_abi in events}
+def decode_event(topic_to_event_abi: Dict[bytes, Dict], log_entry: Dict) -> Dict:
+    event_id = log_entry["topics"][0]
```

Comment:
`event_id` and `topic` look like synonyms. If they are, we should use `topic` to be consistent.

Reply:
Fair point 👀